ui_testing/readme.md

```diff
@@ -9,7 +9,7 @@ cd ui_testing
 # run in head mode
 nosetests-3.4 -vs --logging-level=WARNING --with-flaky --force-flaky --max-runs=3 --no-flaky-report testcases/basic_tests --tc-file=../config.ini --tc=site.password:admin
 # run in headless mode
-xvfb-run -a nosetests-3.4 -vs --logging-level=WARNING  --with-flaky --force-flaky --max-runs=3 --no-flaky-report testcases/basic_tests --tc-file=../config.ini --tc=site.password:admin
+-run -a nosetests-3.4 -vsxvfb --logging-level=WARNING  --with-flaky --force-flaky --max-runs=3 --no-flaky-report testcases/basic_tests --tc-file=../config.ini --tc=site.password:admin
```

Comment:
this is wrong, revert it

Reply:
@0xIslamTaha  Please clarify this comment , Do you mean that I run with nosetests3 and this readme.md file execute from nosetests-3.4